packages/openlmis/src/Utils.js

```diff
@@ -0,0 +1,98 @@
+import { composeNextState } from '@openfn/language-common';
+import {
+  request as commonRequest,
+  logResponse,
+  makeBasicAuthHeader,
+} from '@openfn/language-common/util';
+
+export const authorize = state => {
+  const {
+    baseUrl,
+    access_token,
+    username,
+    password,
+    clientId = 'user-client',
+    clientSecrete = 'changeme',
```

Comment:
This must be a typo on `clientSecret`

Why use the `changeme` pattern and not jsut do `if (!clientSecret)`{ console.log() }`?

Reply:
For user-based credential the default values are 
```
client-id: user-client
secret: changeme
```

Here are the API docs 
- https://docs.openlmis.org/en/rel-3.0.2/api/index.html
- https://github.com/OpenLMIS/openlmis-auth/blob/master/DESIGN.md#auth-service-design

Reply:
I have fixed the typo

Reply:
But default credentials should be in the credential, not in the adaptor? I'm not aware that we've ever hard coded credentials into an adaptor and I'm not sure we should start now

Reply:
Okay i will remove the default and add `clientId` and `clientSecret` in required list in `configuration-schema`

Reply:
Can they not be defaulted in the configuration schema? Json schema has defaults right?

Reply:
Yes i have put the default for both `clientId` and `clientSecret` in `configuration-schema`